book/quick-brick/Setup.md

```diff
@@ -5,57 +5,151 @@
 Then run the following process inside the cloned repo :
 
 ```
 $ yarn zapplicaster prepare -a <app_version_id>
 ```
 
-You can run the script for both your iOS & android version - the QuickBrick workspace allows you to set up simultaneously for one android & one iOS version, without one replacing the other. This is particularly helpful to work simultaneously on a the iOS and android version of a given app.
+The command will retrieve the zapp configuration files for these versions and inject them in the `zapp-react-native-development-app` package.
 
-This command will retrieve the zapp configuration files for these versions and inject them in the `zapp-react-native-development-app` package.
 Please note that if you add / remove plugins, or change the rivers.json configuration in Zapp, you will need to run the prepare command again to get the most recent configuration file locally.
 
-### Prepare your plugin workspace (when creating a plugin)
-
-When creating a plugin, you don't have to work within the Zapp-React-Native repo.
-Simply follow these steps:
-
-- Create a folder for your project
-- Initialize your git repository by running `git init`
-- run `npm init` to initialize your package.json file for your plugin.
-- Run `npx @applicaster/zapplicaster-cli prepare -a <app_version_id> -d .` to bootstrap your plugin's workspace. You can optionaly use the `-y` option to use yarn instead of npm in your project. This will install all the required initial dependencies and add some npm scripts for your project.
-- Add a `main` property to your package.json which points to the file where your plugin's entry point will be. This is the file which will be reached when importing your plugin through npm
-- Add a `.npmignore` file with all the files you don't want to push to npm (like the QuickBrick bootstrap files for instance)
-- Edit `index.js` to import your plugin code, and either push it to the plugin's array, or in the `components` property of the `createZappApp` function call if your plugin is a Ui Builder `ui_component`
-- Manually update the file `config/rivers.json` to insert data which will result in your plugin showing up in the app. If you're not sure how to achieve this, ask #support or a member of the Zapp Team
-
-That's it, you're ready to run your app and start coding !
-
-### run on iOS emulator
+You can run the script for both your iOS & Android versions. The QuickBrick workspace allows you to set up simultaneously for one android & one iOS version, without one replacing the other. This is particularly helpful to work simultaneously on iOS and Android versions of a given app.
+
+### Add a plugin to a QuickBrick workspace
+
+When creating a plugin, follow these steps:
+
+- In your forked [QuickBrick](https://github.com/applicaster/QuickBrick) repository, create a folder for your plugin project. The folder should be located inside the `plugins` directory. 
```

Comment:
I think the guide should start with this. We don’t want to host all plugins, and developers should work on their own qb workspace